src/components-examples/cdk/stepper/cdk-linear-stepper-with-form/example-custom-linear-stepper.html

```diff
@@ -0,0 +1,22 @@
+<section class="example-container">
+    <header>
+        <h2>Step {{ selectedIndex + 1}}/{{ steps.length}}</h2>
+    </header>
+
+    <div [style.display]="selected ? 'block' : 'none'">
```

Comment:
Rather than toggling the content using `display`, you should be able to detach it from the DOM completely using `ngTemplateOutlet`: `[ngTemplateOutlet]="selected ? selected.content : null"`.

Reply:
Also worked.